src/plugins/integrations/pythonRequirementsTxt.ts

```diff
@@ -0,0 +1,27 @@
+import { readFileSync } from 'fs';
+import { SiloDiscoveryConfig } from '../types';
+
+export const pythonRequirementsTxt: SiloDiscoveryConfig = {
+  supportedFiles: ['requirements.txt'],
+  ignoreDirs: ['node_modules', 'serverless-build', 'lambda-build'],
```

Comment:
These don't seem to make sense for python? Would recommend looking up what the build folder is for python (or what is commonly ignored/not checked in to github) and add those here

Reply:
Ok, I'll just remove, I have a few folders that I can try adding